.gitmodules

```diff
@@ -56,6 +56,15 @@
 [submodule "third_party/catch2"]
 	path = third_party/catch2
 	url = https://github.com/catchorg/Catch2.git
+[submodule "third_party/spirv-reflect"]
+	path = third_party/spirv-reflect
+	url = https://github.com/KhronosGroup/SPIRV-Reflect.git
+[submodule "third_party/json"]
+	path = third_party/json
+	url = https://github.com/nlohmann/json.git
```

Comment:
tinygltf already includes a version of this JSON parser.  I understand a possible reluctance to have whatever is depending on this version depend on tinygltf instead, but I'd also be concerned about possible issues if any project depending on both gets the version it wasn't expecting when it tries to include the header.

Reply:
ok, I was mistaken... there are already 2 versions of this parser in the codebase (hwcpipe has one embedded) and this will make 3.